docs/supported_docker_environment/index.md

```diff
@@ -21,7 +21,14 @@ please contact the Testcontainers team and other users from the Testcontainers c
 
 ## Colima
 
-In order to run testcontainers against [colima](https://github.com/abiosoft/colima) the env vars below should be set
+In order to run testcontainers against [colima](https://github.com/abiosoft/colima) the standard configuration should be automatically detected. Starting colima should be enough.
+
+```bash
+colima start --network-address
+docker context use colima
+```
+
+For more complex scenarios you should configure the following environment variables:
```

Comment:
I think these docs can even be removed, to avoid confusion, right?
Same for the Podman and Rancher docs I guess?

Reply:
Happy to further remove the docs.
I think the podman one needs to stay. In my experience, rootless podman installation do not configure the Docker context